content/en/docs/Installation/trustedboot.md

```diff
@@ -121,6 +121,44 @@ title My Awesome OS
 Remember when you build upgrade mediums, to also add your branding, otherwise the new active image will be named differently than your fallback and recovery ones.
 {{% /alert %}}
 
+### Additional efi entries
+
+When booting in UKI mode, the cmdline is part of the signed artifact. This means that in order to pass additional options through the cmdline,
+an additional efi entry is needed. `enki build-uki` supports the following arguments:
+
+- `--extend-cmdline`: Use this option to add options to the cmdline used for active/passive/recovery. This option will not generate additional entries in the systemd-boot menu.
```

Comment:
maybe use "attributes" or "stanzas" instead of the "to add options". 

Also since we are a the extended docs maybe we should also include "auto reset" in the active/passive/recovery. And maybe the best would be to describe that above and call this group of efi files something e.g. Kairos default entries

Reply:
I think the right term is "parameters" (https://www.kernel.org/doc/html/v4.14/admin-guide/kernel-parameters.html). Let me try to re-phrase.

Reply:
Also in the examples below, I think it would be nice to show the list of files generated and the boot menu, that will make it completely clear

Reply:
parameters is good 👍